rfqservice/order.go

```diff
@@ -0,0 +1,313 @@
+package rfqservice
+
+import (
+	"context"
+	"fmt"
+	"sync"
+	"time"
+
+	"github.com/lightninglabs/lndclient"
+	"github.com/lightninglabs/taproot-assets/fn"
+	"github.com/lightninglabs/taproot-assets/rfqmsg"
+	"github.com/lightningnetwork/lnd/lnwire"
+)
+
+// SerialisedScid is a serialised short channel id (SCID).
+type SerialisedScid uint64
+
+// ChannelRemit is a struct that holds the terms which determine whether a
+// channel HTLC is accepted or rejected.
+type ChannelRemit struct {
+	// Scid is the serialised short channel ID (SCID) of the channel to
+	// which the remit applies.
+	Scid SerialisedScid
+
+	// AssetAmount is the amount of the tap asset that is being requested.
+	AssetAmount uint64
+
+	// MinimumChannelPayment is the minimum number of millisatoshis that
+	// must be sent in the HTLC.
+	MinimumChannelPayment lnwire.MilliSatoshi
```

Comment:
+1 as overpay would be accepted, and maybe necessary to handle certain types of price slippage